docs/projects/fmc-website/components/header-help.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Header
+id: header-help
+description: Header Component in Fill my Cycle Website
+sidebar_label: Header
+keywords: [Title, dark-theme, instructions]
+---
+# Header
+The **Header** section is an informative element which is present at the top of every page of the website. It consists of ***Fill My Cycle Header**, **Dark Theme Icon** and a **Interactive Theme changer Logo** making the overall interface of the webiste very convertible. Hence providing essential information and contributing to user-friendly experience. The **"Fill My Cycle"** Header, an artistic portrayal that speaks volumes, engaging the viewer in a captivating dialogue.
+
+### Fill My Cycle Header
+The header is located at the upper centre portion of the page.
+To display the fill my cycle header we need to follow the following steps:
+
+1. Opening the **index.html** file.
+
+2. Inside the file we have firstly created and defined the head section giving it a name also added some tags like `<meta charset="utf-8">` tag tells the computer how to understand and show the characters on the page. Some other tags are The `<meta name="description" content="...">` tag is like a small description of what the page is about. It tells people and search engines a little bit about what they can expect to find on the page and the `<meta name="viewport" content="...">` tag helps make sure the page looks good on different devices like phones or tablets.
+
+3. But in this section our prime focus is to obtain the fill my cycle logo this can be found later in the **index.html** code. Here we created different div classses for classifying different parts of codes and simply used html to type the logo enclosed under `<h3>` tag.
+
+``` html title = "index.html" {3-4} showLineNumbers
+    <div class="container">
+	   <div class="top-container">
+	      <header class="header">
+		      <h3 class="title">Fill My Cycle</h3>  
+``` 
+4. This looks something like this
```

Comment:
Make it something like After completing the above steps our header logo looks like this: and make it bold